_observing-your-data/alerting/dashboards-alerting.md

```diff
@@ -0,0 +1,92 @@
+---
+layout: default
+title: Alerting dashboards and visualizations 
+parent: Alerting
+nav_order: 50
+---
+
+# Alerting dashboards and visualizations
+Introduced 2.9
+{: .label .label-purple }
+
+Create, manage, and take action on your alerts in a single, consolidated view and identify and resolve issues quickly. Use the **Dashboard** interface to:
+
+- Set up, add, and adjust rules and conditions that trigger alerts and notifications.
+- Create graphs that show trends and patterns and build intuitive dashboards to stay informed of important metrics and data points in real time.
+- Monitor your alerts in one place with at-a-glance views.
+
+The following image gives you a snapshot of the Dashboard interface. 
+
+<img src="{{site.url}}{{site.baseurl}}/images/dashboards/alerting-viz.png" alt="Example alerting visualization" width="800" height="800">
```

Comment:
@lezzago Do we also need to update this screenshot (as we've done for AD)?

Reply:
Provided a new image in the `alerting-viz.png` comment.